<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UISimplify</class>
 <widget class="QDialog" name="UISimplify">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>210</width>
    <height>114</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Geometry simplification</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Select theme:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="themeComboBox"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Threshold:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEdit">
     <property name="inputMask">
      <string>00000000000000000; </string>
     </property>
     <property name="text">
      <string>10</string>
     </property>
     <property name="maxLength">
      <number>17</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Output theme:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="text">
      <string>simplified</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>UISimplify</receiver>
   <slot>pushButton_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>64</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>pushButton_clicked()</slot>
 </slots>
</ui>
